runtime-library/utils/type_utils.c

```diff
@@ -254,6 +262,15 @@ get_closure_struct_type(wasm_module_t wasm_module,
         if (field_count_in_ctx != 0) {
             continue;
         }
+        field_type =
+            wasm_struct_type_get_field_type((wasm_struct_type_t)type, 1, &mut);
```

Comment:
Had better define `1` as macro or enum

Reply:
Yeah, we can define an enum to represent field index